### PR TITLE
Countdown Timer using Vanilla JS #17

### PR DESCRIPTION
**Pull request by vishal2005025**
# Pull Request for WebMasterLog 💡

## Issue Title: Countdown Timer using Vanilla JS

- **Name of the project** : Countdown Timer
- **Category:** : Vanila Js
- **Name:** : Vishal Malyan
- **GitHub ID:** : https://github.com/vishal2005025
- **Idenitfy yourself:** : JWOC'24 Contributer

Closes: #17 

### Describe the add-ons or changes you've made 📃

Created a "Countdown Timer" project using HTML ,CSS and Javascript.

## Type of change ☑️

What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested? ⚙️

I verified my project using live server of vsCode

## Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots 📷

1. Screenshot 2024-01-25 181254

## Note to reviewers 📄

Please review my PR and let me know the feedback..Thank You!

### CODE REVIEW

File: Vanilla-JS-Projects/Intermediate/Countdown-Timer/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h3 align='center'>Tech Stack Used 🎮</h3>
+
+<div align='center'>
+
+  ![HTML5](https://img.shields.io/badge/html5-%23E34F26.svg?style=for-the-badge&logo=html5&logoColor=white)
+  ![CSS3](https://img.shields.io/badge/css3-%231572B6.svg?style=for-the-badge&logo=css3&logoColor=white)
+  <img width="55" src="https://raw.githubusercontent.com/gilbarbara/logos/master/logos/javascript.svg"/>
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Countdown-Timer/README.md:11]: Add the standard logo of the JS as i have mentioned.

File: Vanilla-JS-Projects/Intermediate/Countdown-Timer/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+3. <img src='./photo3.webp.webp'>
+
+![Line](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/4b78510f-a941-45f8-a9d5-80ed0705e847)
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h4 align='center'>Developed By <b><i>Vishal Malyan</i></b></h4>
+<p align='center'>
+  <a href='linkedin.com/in/Vishal-malyan'>
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Countdown-Timer/README.md:60]: Link your linkedin link properly with https as in github url

File: Vanilla-JS-Projects/Intermediate/Countdown-Timer/photo1.webp.webp
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Countdown-Timer/photo1.webp.webp:1]: Requirement of screenshot is just 1 with the same name as of directory `Countdown-Timer.webp` only webp format not it's name.

File: Vanilla-JS-Projects/Intermediate/Countdown-Timer/README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+3. Open the project folder in your preferred code editor, now you can view website in live.
+
+</p>
+</div>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h3 align='center'>Screenshot📸</h3>
+1. <img src='./photo1.webp.webp'>
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Countdown-Timer/README.md:48]: Attach these images by uploading in the comment section and add their links here or you can refer from other project's readme files.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@vishal2005025 LGTM, resolve these conflicts and again mentioned to me.

**Comment by vishal2005025** [comment]
sir , please review..

**Review by Avdhesh-Varshney — APPROVED** [review]
@vishal2005025 in single commit you can do all these things which i have mentioned. It's not necessary to do commit for each and every deletion and addition. OK

**Comment by Avdhesh-Varshney** [comment]
@vishal2005025 PR Approved 🎉

**Comment by vishal2005025** [comment]
Sir , sorry for the late reply please change the label of this to # hard as I have work hard and I have also add stopwatch and clock to this issue and spent many days to fix it ..
please change please....

**Pull request merged**